mock2/private/style/scss/contacts/_contacts.scss

```diff
@@ -0,0 +1,36 @@
+.contacts {
+  display: flex;
+  padding: 40px 0;
+  justify-content: space-between;
+
+  &__text {
+    margin-right: 10px;
+    color: $color-text-logo-primary;
+    font-family: "Playfair Display";
```

Comment:
You may also create some `include` rule for repeating font family + other properties like font-weight combinations. Try to find common things in your markup / templates and extract them to simplify your life :)